anonymizer/anonymizer.gemspec

```
@@ -0,0 +1,14 @@
+require_relative "lib/anonymizer/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "anonymizer"
+  spec.version = Anonymizer::VERSION
+  spec.authors = ["RDV Service Public"]
+
+  spec.summary = "Anonymize ActiveRecord data"
```

La raison pour laquelle on anonymise les données (par exemple faire des stats plutôt que d'utiliser des données en démo) explique certains choix de design (pour cet exemple : effacer tous les nous plutôt que de les remplacer par des noms générés aléatoirement)
```suggestion
  spec.summary = "Erase personal data from Postgres tables to use them for stats"
```